packages/components/config/ember-try.js

```diff
@@ -19,18 +19,18 @@ module.exports = async function () {
     },
     scenarios: [
       {
-        name: 'ember-lts-3.24',
+        name: 'ember-lts-3.28',
         npm: {
           devDependencies: {
-            'ember-source': '~3.24.3',
+            'ember-source': '~3.28.0',
```

Comment:
```suggestion
            'ember-source': '~3.28.0',
```
(suggestion) Might be more accurate to use whatever the last point release of 3.28 was

Reply:
sure, will update to 3.28.11